core/client/app/components/gh-profile-image.js

```diff
@@ -0,0 +1,38 @@
+import Ember from 'ember';
+
+export default Ember.Component.extend({
+    email: '',
+    size: 90,
+    hasUploadedImage: true,
+
+    ghostPaths: Ember.inject.service('ghost-paths'),
+    notifications: Ember.inject.service(),
+
+    imageBackground: Ember.computed('hasUploadedImage', 'email', function () {
+        var email = this.get('email'),
+            size = this.get('size'),
+            gravatarUrl;
+
+        if(email && !this.get('hasUploadedImage')) {
+            gravatarUrl = 'http://www.gravatar.com/avatar/' + md5(email) + '?s=' + size + '&d=blank';
+            return 'background-image: url(' + gravatarUrl + ')';
+        } else {
+            return '';
+        }
+    }),
```

Comment:
I think there's a difference between how this property gets computed vs. how it did in the original controller, since there's nothing watching the `image` property of the User model?  Maybe this is what the `hasUploadedImage` property is intended for?